src/components/home/Landing.jsx

```diff
@@ -1,21 +1,62 @@
+"use client";
 import LandingImage from "@/public/images/landing.webp";
 import { FaMapPin } from "react-icons/fa";
 import Button from "@/components/Button";
 import Image from "next/image";
+import { motion } from "framer-motion";
+
+const animations = {
+  start: {
+    opacity: 0,
+    y: 30,
+  },
+  end: {
+    opacity: 1,
+    y: 0,
+  },
+};
 
 const Landing = () => {
   return (
     <div className="flex items-start justify-center my-[5%]">
       <Image src={LandingImage} alt="Landing Picture" />
 
       <div className="absolute z-20 text-isa-white text-nowrap md:text-6xl text-2xl font-bold flex flex-col items-center justify-center mt-[5%]">
-        Indian Student Association
+        <motion.div
+          variants={animations}
+          initial="start"
+          whileInView="end"
+          transition={{ delay: 0.2 }}
+        >
+          Indian Student Association
+        </motion.div>
         <div className="text-isa-white md:text-3xl text-sm flex flex-row">
-          <FaMapPin className="mr-5" />
-          University of California Riverside
+          <motion.div
+            variants={animations}
+            initial="start"
+            whileInView="end"
+            transition={{ delay: 0.2 }}
+          >
+            <FaMapPin className="mr-5" />
```

Comment:
you can put the FaMap Pin in the same motion div in the location

Reply:
you can edit the className in motion.div and use Tailwind CSS to align the pin correctly.